.htaccess

```diff
@@ -0,0 +1,9 @@
+Options +FollowSymLinks
```

Comment:
For dev testing using using the PHP server, this is not necessary and as I mentioned, it is all routed to /examples, you are redirecting all traffic here and it might try to look for /examples in /examples itself again.